rivershared/sqlctemplate/sqlc_template_test.go

```diff
@@ -115,6 +115,53 @@ func TestReplacer(t *testing.T) {
 		`, updatedSQL)
 	})
 
+	t.Run("SyntaxMistakes", func(t *testing.T) {
+		t.Parallel()
+
+		replacer, _ := setup(t)
+
+		// Missing colon after "TEMPLATE".
+		{
+			ctx := WithReplacements(ctx, map[string]Replacement{}, nil)
+
+			_, _, err := replacer.RunSafely(ctx, `
+			-- name: JobCountByState :one
+			SELECT count(*)
+			FROM /* TEMPLATE schema */river_job
+			WHERE state = @state;
+		`, nil)
+			require.EqualError(t, err, "sqlctemplate found template-like tag after replacements; probably syntax error or missing end tag: /* TEMPLATE")
+		}
+
+		// Missing "TEMPLATE_END".
+		{
+			ctx := WithReplacements(ctx, map[string]Replacement{}, nil)
+
+			_, _, err := replacer.RunSafely(ctx, `
+			-- name: JobCountByState :one
+			SELECT count(*)
+			FROM /* TEMPLATE_BEGIN: schema */river_job
+			WHERE state = @state;
+		`, nil)
+			require.EqualError(t, err, "sqlctemplate found template-like tag after replacements; probably syntax error or missing end tag: /* TEMPLATE_BEGIN")
+		}
+
+		// Missing "TEMPLATE_END".
```

Comment:
wrong comment? This one has `TEMPLATE_END`

Reply:
Whoops, that's right. Fixed.